server/lib/schema/helpers/extractMaliciousAnswerValueFromMaliciousQuestionAndJudgeQuestion.ts

```diff
@@ -0,0 +1,49 @@
+const uuidv4 = require("uuid/v4");
+
+export function extractMaliciousAnswerValueFromMaliciousQuestionAndJudgeQuestion(
+  maliciousQuestionValue,
+  judgeQuestionValue,
+) {
+  const honestPointerId = maliciousQuestionValue[0].nodes[3].data.pointerId;
+  const firstPointerId = judgeQuestionValue[0].nodes[3].data.pointerId;
+  const secondPointerId = judgeQuestionValue[0].nodes[5].data.pointerId;
```

Comment:
Feeling a little concerned about these index references (`nodes[5]`). 

How confident are you that these indexes will remain fixed? Maybe add some checks here.

Also would appreciate a comment in the code documenting why you are accessing these particular indexes

Reply:
The malicious and judge question values are both constructed programmatically, so these index references will be OK as long as that question value generation code stays the same. 

I agree it's a form of coupling that doesn't feel great. I'll add some checks and leave a comment.